src/main/scala/com/lightbend/conductr/sbt/BundlePlugin.scala

```diff
@@ -395,93 +424,81 @@ object BundlePlugin extends AutoPlugin {
-        formatValue(s"""    endpoints = %s""", toString((endpointsConfigName in config).value, (v: Map[String, Endpoint]) => formatEndpoints(bundleConfVersion.value, v))) ++
-        Seq("  }", "}") ++
-        checkComponents
-
-    declarations.mkString("\n")
-  }
-
-  private def stageBundle(config: Configuration, bundleTypeConfig: Configuration): Def.Initialize[Task[File]] = Def.task {
-    validateSettings(config).value
-
-    val bundleTarget = (NativePackagerKeys.stagingDirectory in config).value / config.name
-    writeConfig(bundleTarget, (bundleConf in config).value)
-    val componentTarget = bundleTarget / (normalizedName in config).value
-    IO.copy((mappings in bundleTypeConfig).value.map(p => (p._1, componentTarget / p._2)))
-    componentTarget
-  }
-
-  private def stageConfiguration(config: Configuration): Def.Initialize[Task[File]] = Def.task {
-    validateSettings(config).value
-
-    val configurationTarget = (NativePackagerKeys.stagingDirectory in config).value / config.name
-    val generatedConf = (bundleConf in config).value
-    val srcDir = (sourceDirectory in config).value / (configurationName in config).value
-    if (generatedConf.isEmpty && !srcDir.exists())
-      sys.error(
-        s"""Directory $srcDir does not exist.
-            | Specify the desired configuration directory name
-            |  with the 'configurationName' setting given that it is not "default"""".stripMargin
-      )
-    IO.createDirectory(configurationTarget)
-    if (generatedConf.nonEmpty) writeConfig(configurationTarget, generatedConf)
-    IO.copyDirectory(srcDir, configurationTarget, overwrite = true, preserveLastModified = true)
-    configurationTarget
-  }
+    if (hasValue((normalizedNameConfigName in config).value) ||
+      hasValue((compatibilityVersionConfigName in config).value) ||
+      hasValue((systemConfigName in config).value) ||
+      hasValue((systemVersionConfigName in config).value) ||
+      hasValue((nrOfCpusConfigName in config).value) ||
+      hasValue((memoryConfigName in config).value) ||
+      hasValue((diskSpaceConfigName in config).value) ||
+      hasValue((rolesConfigName in config).value) ||
+      hasValue((projectInfoConfigName in config).value) ||
+      hasValue((bundleTypeConfigName in config).value) ||
+      hasValue((startCommandConfigName in config).value) ||
+      hasValue((endpointsConfigName in config).value)) {
+
+      val checkComponents = (checksConfigName in config).value match {
+        case (value, configName) if (forAllSettings || configName.isDefined) && value.nonEmpty =>
+          val checkInitialDelayValue = (checkInitialDelay in config).value
+          val checkInitialDelayInSeconds =
+            if (checkInitialDelayValue.toMillis % 1000 > 0)
+              checkInitialDelayValue.toSeconds + 1 // always round up
+            else
+              checkInitialDelayValue.toSeconds
+          Seq(
+            value.map(uri => s""""$uri"""").mkString(
```

Comment:
It is surrounding the value with double quotes. Changed in order to make this clearer:

```
            value.map(uri => s"\"$uri\"").mkString(
```